docs/en/guides/sre/scaling-clusters.md

```diff
@@ -9,14 +9,14 @@ description: ClickHouse does not support automatic shard rebalancing, so we prov
 
 ClickHouse does not support automatic shard rebalancing. However, there are ways to rebalance shards in order of preference:
 
-1. Adjust the shard for the [distributed table](../../engines/table-engines/special/distributed/), allowing writes to be biased to the new shard. This potentially will cause load imbalances and hot spots on the cluster but can be viable in most scenarios where write throughput is not extremely high. It does not require the user to change their write target i.e. It can remain as the distributed table. This does not assist with rebalancing existing data.
+1. Adjust the shard for the [distributed table](../../../engines/table-engines/special/distributed/), allowing writes to be biased to the new shard. This potentially will cause load imbalances and hot spots on the cluster but can be viable in most scenarios where write throughput is not extremely high. It does not require the user to change their write target i.e. It can remain as the distributed table. This does not assist with rebalancing existing data.
```

Comment:
```suggestion
1. Adjust the shard for the [distributed table](/docs/en/engines/table-engines/special/distributed.md), allowing writes to be biased to the new shard. This potentially will cause load imbalances and hot spots on the cluster but can be viable in most scenarios where write throughput is not extremely high. It does not require the user to change their write target i.e. It can remain as the distributed table. This does not assist with rebalancing existing data.
```

Reply:
Full paths from `/docs` seem to work best, and including `.md` causes the Docusaurus build to test the link